www/scipylib/building/windows.rst

```diff
@@ -32,6 +32,109 @@ information than this document will provide.
 .. _OpenBLAS: https://github.com/xianyi/OpenBLAS
 .. _LAPACK: http://www.netlib.org/lapack/
 
+
+Building the Released SciPy
+---------------------------
+
+This section provides the step-by-step process to build the released scipy. If you want
+to build completely from source, you should estimate at least three hours to build all
+libraries and compile SciPy. Feel free to stop and inspect any step at any time, but
```

Comment:
Maybe rephrase this - if people do run into trouble this sentence will make them feel particularly bad.